xml/System.Collections.ObjectModel/ObservableCollection`1.xml

```diff
@@ -461,8 +461,8 @@
         <ReturnType>System.Collections.Specialized.NotifyCollectionChangedEventHandler</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Occurs when an item is added, removed, changed, moved, or the entire list is refreshed.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Occurs when an item is added, removed, changed, or moved, or the entire list is refreshed.</summary>
+        <remarks>Does not fire when a property of a collection item changes.</remarks>
```

Comment:
Wouldn't this suffice?

```suggestion
        <summary>Occurs when an item is added, removed, moved, or the entire list is refreshed.</summary>
        <remarks>To be added.</remarks>
```

Reply:
My understanding was that the "changed" wording referred to a situation such as:
```
collection[0] = item1;
collection[0] = item2;
```
But that could be considered an Add and a Remove even though `.Add()` and `.Remove()` aren't used.

Reply:
I took the wording from description of the type itself: https://docs.microsoft.com/en-us/dotnet/api/system.collections.objectmodel.observablecollection-1?view=net-6.0#definition

I don't think a clarification on mutation of individual collection items is warranted, for generic collections it is generally not possible to achieve anyway.